airbyte-cdk/python/bin/generate-component-manifest-dagger.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -e
+
+pip install dagger-io==0.9.6
+python bin/generate_component_manifest_files.py
```

Comment:
@alafanechere do you think it would be clearer to add `dagger-io` as a dev dependency, and then we can call `airbyte-cdk/python/bin/generate_component_manifest_files.py` from the pyproject.toml file?